_posts/2019-01-01-linguistics-minute-saxon-anglian-spelling.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Linguistics Minute: Saxon & Anglian Spelling"
+date: "2019-01-01 12:00:00 -0500"
+---
+_This was originally published in my work Slack on 2016-10-03, but I've since
+been convinced that more people than just those few who can even read it there
+might be interested. I've edited it for clarity and correctness. I'm going to
+migrate all my Linguistcs Minutes over to my blog and probably post future ones
+here._
+
+It turns out that spelling was basically always fucked in English. When the
+Germanic invaders of Britain first started writing, there were basically two
```

Comment:
Maybe note when the Germanic invasions happened?

Reply:
I don't actually know. Heh.